src/patternfly/demos/LoginPage/login-page.hbs

```diff
@@ -1,6 +1,6 @@
-<div class="pf-d-login-page {{pf-d-login-page--modifier}}"
-  {{#if pf-d-login-page__id}}
-    id="{{pf-d-login-page__id}}"
+<div class="pf-d-login-page {{#if login-page--modifier}}{{login-page--modifier}}{{/if}}"
```

Comment:
remove space after pf-d-login-page and add it before {{login-page--modifier}}

Reply:
fixed